src/main/java/com/github/susom/database/DatabaseImpl.java

```diff
@@ -27,9 +28,6 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-/*>>>
-import org.checkerframework.checker.tainting.qual.Untainted;
-*/
 
```

Comment:
I don't think you want to remove this, as it is used by the static analysis tool (Checker Framework).